src/ResourceManager/Profile/Commands.Profile.Test/LongRunningCmdletTests.cs

```diff
@@ -48,7 +48,6 @@ public LongRunningCmdletTests(ITestOutputHelper output)
         }
 
         [Fact]
-        [Trait(Category.AcceptanceType, Category.CheckIn)]
```

Comment:
are all of these flaky?  I think only 2 or 3 of these actually fail (the *Exception* ones), but if not, this is fine.

Reply:
@markcowl I've found jobs where all of the below tests, with the exception of `CanHandleCmdletStop` and `JobCopiesCmdletParameterSet`, have failed at least one recent build, so I disabled all tests in the class for investigation.